docs/configure-upstream-collector.md

```diff
@@ -0,0 +1,13 @@
+# Configure an upstream collector
+
+You can configure an upstream collector, like a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution for the collector, to collect logs and metrics and send them to Elastic Observability.
+
+<!-- We might want to add some pros and cons of using an upstream collector -->

```

Comment:
@mlunadia would it be valuable to layout some pros and cons of using an upstream collector vs. the elastic distro?